packages/report-server/src/routes/FetchReportRoute.ts

```diff
@@ -63,6 +82,19 @@ export class FetchReportRoute extends Route<FetchReportRequest> {
     };
 
     const reportBuilder = new ReportBuilder(reqContext).setConfig(report.config);
-    return reportBuilder.build();
+
+    const reportResponse = await reportBuilder.build();
+
+    const { results } = reportResponse;
+
+    // If the report has more than one result, update the report.latest_data_parameters with the request parameters
+    if (this.getHasData(results)) {
+      await report.setLatestDataParameters({
+        projectCode: hierarchy,
+        entityCodes: reportQuery.organisationUnitCodes,
```

Comment:
I'd be open to just leaving the parameter names as is `hierarchy` and `organisationUnitCodes`. That way we can just directly feed the parameters back into the `/fetchReport` route and get data back?

If we had wanted to change those parameter names I reckon we can do it at the api level itself :+1: